CHANGELOG.md

```diff
@@ -259,33 +272,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-[3.0.0]: https://github.com/MetaMask/controllers/compare/v2.0.5...v3.0.0
-[2.0.5]: https://github.com/MetaMask/controllers/compare/v2.0.4...v2.0.5
-[2.0.4]: https://github.com/MetaMask/controllers/compare/v2.0.3...v2.0.4
-[2.0.3]: https://github.com/MetaMask/controllers/compare/v2.0.2...v2.0.3
-[2.0.2]: https://github.com/MetaMask/controllers/compare/v2.0.1...v2.0.2
-[2.0.1]: https://github.com/MetaMask/controllers/compare/v2.0.0...v2.0.1
-[2.0.0]: https://github.com/MetaMask/controllers/releases/tag/v2.0.0
+[Unreleased]: git+https://github.com/MetaMask/controllers/compare/v10.2.0...HEAD
+[10.2.0]: git+https://github.com/MetaMask/controllers/compare/v10.1.0...v10.2.0
+[10.1.0]: git+https://github.com/MetaMask/controllers/compare/v10.0.0...v10.1.0
+[10.0.0]: git+https://github.com/MetaMask/controllers/compare/v9.1.0...v10.0.0
+[9.1.0]: git+https://github.com/MetaMask/controllers/compare/v9.0.0...v9.1.0
+[9.0.0]: git+https://github.com/MetaMask/controllers/compare/v8.0.0...v9.0.0
+[8.0.0]: git+https://github.com/MetaMask/controllers/compare/v7.0.0...v8.0.0
+[7.0.0]: git+https://github.com/MetaMask/controllers/compare/v6.2.1...v7.0.0
+[6.2.1]: git+https://github.com/MetaMask/controllers/compare/v6.2.0...v6.2.1
+[6.2.0]: git+https://github.com/MetaMask/controllers/compare/v6.1.1...v6.2.0
+[6.1.1]: git+https://github.com/MetaMask/controllers/compare/v6.1.0...v6.1.1
+[6.1.0]: git+https://github.com/MetaMask/controllers/compare/v6.0.1...v6.1.0
+[6.0.1]: git+https://github.com/MetaMask/controllers/compare/v6.0.0...v6.0.1
+[6.0.0]: git+https://github.com/MetaMask/controllers/compare/v5.1.0...v6.0.0
+[5.1.0]: git+https://github.com/MetaMask/controllers/compare/v5.0.0...v5.1.0
+[5.0.0]: git+https://github.com/MetaMask/controllers/compare/v4.2.0...v5.0.0
+[4.2.0]: git+https://github.com/MetaMask/controllers/compare/v4.1.0...v4.2.0
+[4.1.0]: git+https://github.com/MetaMask/controllers/compare/v4.0.2...v4.1.0
+[4.0.2]: git+https://github.com/MetaMask/controllers/compare/v4.0.1...v4.0.2
+[4.0.1]: git+https://github.com/MetaMask/controllers/compare/v4.0.0...v4.0.1
+[4.0.0]: git+https://github.com/MetaMask/controllers/compare/v3.2.0...v4.0.0
+[3.2.0]: git+https://github.com/MetaMask/controllers/compare/v3.1.0...v3.2.0
+[3.1.0]: git+https://github.com/MetaMask/controllers/compare/v3.0.1...v3.1.0
+[3.0.1]: git+https://github.com/MetaMask/controllers/compare/v3.0.0...v3.0.1
+[3.0.0]: git+https://github.com/MetaMask/controllers/compare/v2.0.5...v3.0.0
+[2.0.5]: git+https://github.com/MetaMask/controllers/compare/v2.0.4...v2.0.5
+[2.0.4]: git+https://github.com/MetaMask/controllers/compare/v2.0.3...v2.0.4
+[2.0.3]: git+https://github.com/MetaMask/controllers/compare/v2.0.2...v2.0.3
+[2.0.2]: git+https://github.com/MetaMask/controllers/compare/v2.0.1...v2.0.2
+[2.0.1]: git+https://github.com/MetaMask/controllers/compare/v2.0.0...v2.0.1
+[2.0.0]: git+https://github.com/MetaMask/controllers/releases/tag/v2.0.0
```

Comment:
These changes are not desired. This is caused by the `repository` field in `package.json`. Unfortunately `auto-changelog` only supports HTTPS repository URLs, so we've been updating the manifest to use the HTTPS url in all repos to fix this issue.